python/cuspatial/cuspatial/demos/hausdorff_distance_verify_locust256.py

```diff
@@ -16,94 +16,101 @@
 import cuspatial.bindings.spatial as gis
 import cuspatial.bindings.soa_readers as readers
 
-data_dir="/home/jianting/trajcode/"
-data_set="locust256"
+data_dir = "/home/jianting/trajcode/"
```

Comment:
Let's remove personal home directories from our released code. We need to figure out a dataset that's small enough (a few kb) to ship with cuspatial for testing.